conference.js

```diff
@@ -305,6 +307,10 @@ class ConferenceConnector {
     _onConferenceFailed(err, ...params) {
         APP.store.dispatch(conferenceFailed(room, err, ...params));
         logger.error('CONFERENCE FAILED:', err, ...params);
+        const state = APP.store.getState();
+
+        // The conference we have already joined or are joining.
+        const conference_ = getCurrentConference(state);
```

Comment:
Why `conference_`? I don't think we have variables with `_` at the end somewhere in the code. Is this some kind of code convention? What does it mean?

Reply:
Talked to Lyubo and thats done in order to avoid a duplicate name with another variable that has the same name.  When I was trying to find out conferenceWillLeave worked i just copied it as it appeared there but the `_` has no use here so it shall be removed

Reply:
- [ ] remove _